docs/examples/dl3_tool_config.json

```diff
@@ -12,7 +12,14 @@
   },
   "DL3FixedCuts": {
```

Comment:
This is called "FixedCuts" and now contains both the global cut value and the values used to calculate the energy dependent cuts at the same time. How is it determined, what is used?

Reply:
I named it, as we use either a _fixed_ global cut or energy-dependent cuts, with _fixed_ efficiency. If this is flawed logic, or is misleading, I will rename it to `DL3Cuts`?

Reply:
In the first case, it's a property of the cut itself, in the other case it is a property of how the cuts are determined. So yes, I think better remove the "Fixed".

Reply:
Understood. Thanks!